app/models/hackathon/digest/listing.rb

```diff
@@ -1,4 +1,5 @@
 class Hackathon::Digest::Listing < ApplicationRecord
   belongs_to :digest, class_name: "Hackathon::Digest"
   belongs_to :hackathon
+  belongs_to :subscription
```

Comment:
A `Listing` links a `Hackathon` to a `Digest`. A `Hackathon::Subscription` centers around location, hence `Listings::ByLocation`. I don't think we should be adding a location subscription here, since that's about the mechanisms of which listings are generated.

Reply:
For the digest, it would be ideal to split the list of hackathons by location. In other words, group the list of `Hackathon`s by their `Subscription`. Having `Subscription` on `Listing` will allow us to do this.

In addition, it's nice to be able to audit which `Listing`s a `Subscription` has been responsible for creating.